.cirrus/publish-macos-silicon.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -ex
+
+brew install sbt
```

Comment:
Not sure if JDK is available, but if so I'd avoid Home Brew (this can be done post-merge).

```suggestion
RUNNER_VER=1.8.2
mkdir -p $HOME/bin
export PATH="$HOME/bin:$PATH"
curl -sL https://raw.githubusercontent.com/sbt/sbt/v$RUNNER_VER/sbt > $HOME/bin/sbt
```

Reply:
It seems there is no JDK by default on the image and `brew install sbt` installs both sbt and a JDK. What is the issue with using `brew`?

Reply:
Generally I avoid it since we don't control it, and it installs a JDK as opposed to us picking a JDK explicitly. For the purpose of sbtn it probably doesn't matter much since we are launching GraalVM but normally we need to use JDK 8.